api/server.go

```
@@ -21,7 +23,12 @@ func (s *Server) Start() {
 	e := s.r
 	e.Use(middleware.RequestID())
 
-	e.POST("/", s.createOrder)
+	g := e.Group("/api/orders/v1.0")
+
+	g.POST("/", s.createOrder)
+
+	slog.Info("Server started on port", "Port", s.port)
 
 	e.Start(":" + s.port)
```

golangci-lint (1.62.2)

Error return value of `e.Start` is not checked

(errcheck)